packages/global/config/omeda-identity-x.js

```diff
@@ -60,4 +60,21 @@ module.exports = ({
   appendPromoCodeToHook,
   appendBehaviorToHook,
   appendDemographicToHook,
+  onLoginLinkSentFormatter: (async ({ req, payload }) => {
+    const omeda = req.app.locals.site.getAsObject('omeda');
+    const { source } = req.body;
+    const { onLoginLinkSentPromoCodes } = omeda;
```

Comment:
It would be cleaner IMO to just pass these values to this file directly (from the site omeda-idx cfg), rather than doing a deep retrieval for them. They are specific to the Omeda+IdX integration anyway.